my-app/src/chatWindow/chatWindow.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import "./styles/chatWindow.scss";
+import { PersonsAside } from "./personsAside/personsAside.js";
+import { PERSONS } from "./constants/chatConstants.js";
+import { ChatBody } from "./chatBody/chatBody.js";
+import { personsContext } from "./personsAside/personsContext.js";
+
+import { messagesService } from "./servises/chatApiService.js";
+
+export function ChatWindow() {
+	// the code below is required for the default selection of the interlocutor
+	const [selectedPerson, setSelectedPerson] = React.useState([
+		PERSONS[0].FIRST_NAME,
+		PERSONS[0].SECOND_NAME,
+		PERSONS[0].AVATAR,
+	]);
+
+	// the code below simulates a request to the server and receiving a response (in the form of sending a message and receiving a message)
+	const [allMessages, setAllMessage] = React.useState([]);
+
+	const handleStartChat = async (e) => {
+		let messages = [];
+		if (e !== "") {
+			await messagesService.getMyMessage(e);
```

Comment:
this is because messages sent and received are stored in the same array. It is not required to return an array after sending a message. An entire array is returned after receiving the message.